internal/featuretests/replaceprefix_test.go

```diff
@@ -153,10 +150,7 @@ func basic(t *testing.T) {
 			envoy_v2.RouteConfiguration("ingress_http"),
 		),
 		TypeUrl: routeType,
-	}).Status(vhost).Equals(contour_api_v1.HTTPProxyStatus{
-		CurrentStatus: k8s.StatusInvalid,
-		Description:   "duplicate replacement prefix '/foo'",
-	})
+	}).Status(vhost).HasError("PrefixReplaceError", "PrefixReplaceError", "duplicate replacement prefix '/foo'")
```

Comment:
If we have `PrefixReplaceError` as the type, then we can have separate reasons for the different error cases (ambiguity, duplicates).